### PR TITLE
[HackerRank]  Simple Array Sum

### PR DESCRIPTION
**Pull request by samit22**
Added simple sum of array numbers in ruby.
Problem: https://www.hackerrank.com/challenges/simple-array-sum/problem
Paste the following code snippet
```ruby
#!/bin/ruby

#
# Complete the simpleArraySum function below.
#
def simpleArraySum(numbers)
  num = numbers.split(" ")
  num.inject(0) { |sum, i| sum + i.to_i }
end

fptr = File.open(ENV['OUTPUT_PATH'], 'w')

ar_count = gets.to_i

ar = gets.rstrip

result = simpleArraySum(ar)

fptr.write result
fptr.write "\n"

fptr.close()

```

### CODE REVIEW

**Comment by aneels3** [comment]
Hello @samit22 please mention the exact problem name as given in the HackerRank.

**Comment by samit22** [comment]
@aneels3 Here's the link:
https://www.hackerrank.com/challenges/simple-array-sum/problem

**Comment by aneels3** [comment]
hello @samit22 please provide a working snippet of the code. Your code should run in the HackerRank platform.

**Comment by samit22** [comment]
Updated @aneels3 👍

**Comment by aneels3** [comment]
@samit22 change the code and commit the changes.

**Comment by aneels3** [comment]
You should contribute the full runnable code. See other PRs

File: HackerRank Problem/Ruby/add_array_sum.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ar_count = gets.to_i
+
+ar = gets.rstrip
+
+result = simpleArraySum(ar)
+
+fptr.write result
+fptr.write "\n"
+
+fptr.close()
```
Comment (aneels3) [inline HackerRank Problem/Ruby/add_array_sum.rb]: delete the extra spaced lines after each statements

Comment (samit22) [reply HackerRank Problem/Ruby/add_array_sum.rb]: That's how hackerrank has it on their site. Removed it though.

Comment (aneels3) [reply HackerRank Problem/Ruby/add_array_sum.rb]: Yeah, but that's not a good coding practice. You can see the documentation of any language.

**Comment by aneels3** [comment]
@samit22 Great Work!

**Pull request merged**